server/specs/db.test.js

```diff
@@ -0,0 +1,27 @@
+const db = require('../../database/index.js');
+
+describe('DB', () => {
+  test('Should store a new booking date to the room user has selected', (done) => {
+    let testData = {
+      id: 21,
+      booked: '2018-02-20',
+      guest_name: 'Eric',
+    };
+
+    db.update(testData);
+
+    db.findOne(testData.id, (data) => {
+      let room = data;
+      expect(room.booked_dates[room.booked_dates.length - 1]).toBe('2018-02-20');
+    });
+    done();
```

Comment:
make sure to also delete this test data before you do done

Reply:
@jennqiao Sure I'll add it to Trello ticket